{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/wagtailapp/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Could we make some render tests for the the custom blocks here? Just to show how it's done